### ISSUE TITLE
Body does not have an attribute is_hfss

### ISSUE THREAD

**Issue by StevenTouzard** [leghtas/HFSSdrawpy#44]
A lot of methods in drawpylib require the Body from drawpy to have an attribute is_hfss. However, Modeler is not initialized with is_hfss. A lot of scripts I tried do:

pm = Modeler('hfss')
pm.is_hfss = True
chip = Body(pm, 'chip')

In this case chip doesn't inherit is_hfss which causes problems later. I can workaround by defining is_hfss for each Body but it doesn't seem like that was the goal. 

Should we just initialize the Modeler with this attribute ?

**Comment by rlescanne** [comment]
This is correct. This property is a reminiscent feature from the past version of the code. We need to modify HFSSdrawpy so that : 
- either the body inherits the modeler property "is_hfss"
- or replace each time we are testing for the chip.is_hfss property by testing on modeler.is_hfss property. 
I think the first one will be more transparent.
